presto-product-tests/conf/presto/etc/catalog/mysql.properties

```diff
@@ -1,4 +1,4 @@
 connector.name=mysql
-connection-url=jdbc:mysql://mysql:3306/test
+connection-url=jdbc:mysql://mysql:13306/test
```

Comment:
port 13306 only works outside docker.  Travis and run_on_docker.sh both run product tests inside a docker container, so this change will break them.